app.js

```diff
@@ -1,18 +1,15 @@
-require('dotenv').config();
+require("dotenv").config();
 
-const express = require('express');
-const hbs = require('hbs');
-
-// require spotify-web-api-node package here:
+const express = require("express");
+const hbs = require("hbs");
 
 const app = express();
 
-app.set('view engine', 'hbs');
-app.set('views', __dirname + '/views');
-app.use(express.static(__dirname + '/public'));
-
-// setting the spotify-api goes here:
+app.set("view engine", "hbs");
+app.set("views", __dirname + "/views");
+app.use(express.static(__dirname + "/public"));
 
-// Our routes go here:
+const routes = require("./config/routes.config");
+app.use(routes);
 
-app.listen(3000, () => console.log('My Spotify project running on port 3000 🎧 🥁 🎸 🔊'));
+app.listen(3000, () => console.log("My Spotify project running on http://localhost:3000 🎧 🥁 🎸 🔊"));
```

Comment:
```suggestion
const port = process.env.PORT || 3000
app.listen(port, () => console.log(`My Spotify project running on http://localhost:${port} 🎧 🥁 🎸 🔊`));
```
Pequeño cambio de configuracion xD